src/styles/components/_component-card.scss

```diff
@@ -1,96 +1,94 @@
-.component-link {
-  display: block;
-  outline: none;
+.component-card {
+  display: flex;
+  flex-direction: column-reverse;
+  height: 21.875rem; // 350px/16
   width: 100%;
-  height: 100%;
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  border: 2px solid var(--foreground-color);
+  background-color: var(--background-color);
 
-  .component-card {
-    display: flex;
-    flex-direction: column-reverse;
-    height: 21.875rem; // 350px/16
-    width: 100%;
-    cursor: pointer;
+  @media (min-width: $breakpoint-scale-up) {
+    height: 100%;
+  }
+
+  @media (min-width: $breakpoint-scale-up) {
+    font-size: map.get($font-sizes, "sm");
+  }
+
+  &__image-open {
+    display: block;
+    box-sizing: border-box;
     transition: all 150ms ease-in-out;
-    border: 2px solid var(--foreground-color);
-    background-color: var(--background-color);
+    width: 100%;
+    height: 100%;
+    padding: min(15%, 2rem);
+    margin: auto;
+
+    @media (prefers-color-scheme: dark) {
+      filter: invert(1);
+    }
 
-    &__image {
+    .component-link:hover & {
+      display: none;
+    }
+  }
+
+  &__image-closed {
+    display: none;
+
+    .component-link:hover & {
+      display: block;
       box-sizing: border-box;
       transition: all 150ms ease-in-out;
       width: 100%;
-      height: 70%;
+      height: 100%;
```

Comment:
I think we still need the `height: 70%` on both the `&__image-open`/`&__image-closed`. With these now at 100%, we're squishing the card headings.